packages/storage-mixin/lib/backends/splice-disk.js

```diff
@@ -27,19 +27,17 @@ function SpliceDiskBackend(options) {
 
   // patch the serialize methods in both backends
   var condition = options.secureCondition;
-  DiskBackend.prototype.serialize = function(model) {
```

Comment:
I might be missing something, where do we use `splice-disk`? I see that connection storage uses `splice-disk-ipc`.
I'm wondering if this can possibly be removed?

Reply:
> I might be missing something, where do we use `splice-disk`?

In my upcoming PR(s?) for connection import/export through the CLI :) Not that I planned for that, I also thought about removing this, but it’s working and will be useful later this week so I’d like to keep it for now.

(There might be an argument for sharing more code between this and splice-disk-ipc but I also don’t want to put a lot of effort into making storage-mixin prettier right now.)